.NET/Microsoft.Recognizers.Text/Recognizer.cs

```diff
@@ -17,12 +19,12 @@ protected void RegisterModel(string culture, string options, Dictionary<Type, IM
             modelContainer.RegisterModel(culture, models, options);
         }
 
-        public IModel GetModel<TModel>(string culture, bool fallbackToDefaultCulture = true, string options = "")
+        public IModel GetModel<TModel>(string culture, bool fallbackToDefaultCulture = true, string options = NoneOptions)

```

Comment:
No related unit test?

Reply:
I'll come up with a way to add these. As they will not follow the current Spec format.